src/include/sof/audio/format.h

```diff
@@ -3,8 +3,8 @@
  * Copyright(c) 2016 Intel Corporation. All rights reserved.
  *
  * Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
- *         Liam Girdwood <liam.r.girdwood@linux.intel.com>
- *         Keyon Jie <yang.jie@linux.intel.com>
+ *			  Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *			  Keyon Jie <yang.jie@linux.intel.com>
```

Comment:
@ShriramShastry I think your editor was trying to be too helpful and converted some spaces to tabs and vice versa. Probably worth switching it off. Fwiw, if you use ```git gui``` you can manually pick the lines you want to commit and then discard the rest that are not needed (useful when your editor is being too helpful!).